src/components/Auth/ChangePasswordForm.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useRef, Fragment, useContext } from 'react';
+import axios from "axios";
+
+import Card from '../UI/Card/Card';
+import classes from './Auth.module.css';
+import Button from '../UI/Button/Button';
+import AuthContext from "../../store/AuthContext";
+
+const ChangePasswordForm = () => {
+    const [formIsValid, setformIsValid] = useState(false);
+    const [isShowForm, setShowForm] = useState(true);
+    const [message, setMessage] = useState(null);
+
+    const authCtx = useContext(AuthContext);
+    const token = authCtx.token;
+
+    const passwordInput = useRef();
+
+    const checkFormValidHandler = () => {
+        if (passwordInput === "") {
+            setformIsValid(false);
+            return;
+        }
+        setformIsValid(true);
+    }
+
+    const submitHandler = (e) => {
+        e.preventDefault();
+        if (!formIsValid) {
+            return;
+        }
+        const changePwAPI = `http://localhost:5151/change_password`;
+        console.log(passwordInput.current.value);
+        axios.post(changePwAPI,
+            {
+                "new_password": passwordInput.current.value
+            },
+            {
+                headers: {
+                    "Authorization": token
+                },
+            }
+        )
+            .then(response => {
+                if (response.status == 200) {
+                    setMessage("Change password successfully");
+                    setShowForm(false);
+                    return;
+                }
+                setMessage("Change password failed, please check information!");
+            })
+            .catch((error) => {
+                setMessage(error.message);
+            });
+    }
```

Comment:
In the chappter https://gitlab.com/pav-training-program/advance-programs/-/blob/reactjs/reactjs/setup-environment.md, you installed Prettier Code Formatter, so please use keystrokes `Shift + Alt + F`  to format your source code.